### PR TITLE
New mining turf sprites

### LINKED ISSUE #13256: Lava can be burned/damaged to look like plating
Reminder for myself to fix this

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
greyt

lava
From @WJohn 

The sandy asteroid floor has been redone to match the new grey walls because

A) Neither of us really liked the red walls in the first place

B) It's a lot less work code wise to have a single set of mining wall sprites

C)Grey walls fit for both volcanic settings, as well as classic grey asteroids.

Also new smooth lava

With sprites from @WJohn

Old lava is still in because this stuff only really looks good with lavaworld asteroid tiles

Fixes #13256

Updated mining turfs to be more modular

### CODE REVIEW

**Comment by Incoming5643** [comment]
It looks a lot like alien creep, also bright red asteroid walls was always "too old to change" territory to me.

**Comment by Anonus** [comment]
They look pretty good yup, but a bit like alien creep.

**Comment by KorPhaeron** [comment]
I am unfamiliar with alien creep

**Comment by Incoming5643** [comment]
You know, the creep from aliens, alien creep

**Comment by Bawhoppen** [comment]
make it like a greyer stone

**Comment by Incoming5643** [comment]
specifically yeah that's what's bothering me, it's so dark my mind is telling me "that's on the floor, not a wall"

**Comment by KorPhaeron** [comment]
Oh the weeds.

Shit it does look like the weeds. Shoulda known better than to trust WJ to not sneak aliens into the PR

**Comment by MMMiracles** [comment]
As a note if you're gonna be replacing the flooring, all the wall's baseturfs on levels 1-3 are basalt, while the 4th level is lava.

**Comment by Zilenan91** [comment]
Now for WJ to enact his grand plans of making mining mobs able to be facehugged

**Comment by KorPhaeron** [comment]
@MMMiracles 

Your mission causes a ton of lag by the way

**Comment by MMMiracles** [comment]
anything i make is perfect thats impossible

**Comment by Tokiko1** [comment]
The inside of the walls would probably look better if there was fewer holes. I feel that all of these details make it look more like the rough surface of a stone floor rather than the inside of a wall. Compare with the station walls or the old mining rock. Just a personal opinion though.

**Comment by Fox-McCloud** [comment]
Bay's asteroid sprites are also pretty damn good.

**Comment by Anonus** [comment]
Yeah tbh the walls shouldn't be lighter than the floor it confuses the eye.

**Comment by Niknakflak** [comment]
Look like alien weeds yea, just agreeing with everyone else here

**Comment by Cheridan** [comment]
maybe its just me, but these walls look kinda like the alien weeds

**Comment by Zilenan91** [comment]
I dunno guys, these walls look a bit like weeds

**Comment by Bawhoppen** [comment]
Alien Hunter (420) has planted some weeds!

**Comment by WJohn** [comment]
Hm, I designed these walls to appear like basalt. Normally I'd have made them very dark like the floors, but that wasn't much of an option for lava world as I need contrast to tell walls and floors apart. I couldn't exactly somehow make them darker than the floor, either, as basalt floor is already nearly totally black.

**Comment by Zilenan91** [comment]
The problem is that they have so many holes in them, rather than them being solid like the current rock

**Comment by WJohn** [comment]
I could probably tone down the amount of holes in it, I'll do that in a bit. Easy fix.

**Comment by xxalpha** [comment]
I prefer black asteroid.

:+1:

**Comment by bgobandit** [comment]
how do the ore overlays look on them

**Comment by tkdrg** [comment]
also note that these new walls aren't smoothing properly when right next to a lava floor

**Comment by Anonus** [comment]
They will smooth properly when next to lava, that's just an artifact of how lava was done before for lavaworld, I'm pretty sure.

**Comment by KorPhaeron** [comment]
That was a mockup of me just spawning turfs so the mineral overlays failed to update

**Comment by phil235** [comment]
:+1:

**Comment by WJohn** [comment]
http://www.mediafire.com/download/90q0n055cz691wy/VOLCANO_LAND_1.1.3b.rar

New rock wall sprites in volcano world.dmi if you want to add them, I made em less porous than our current ones so it won't look quite so much like alien resin.

**Comment by KorPhaeron** [comment]
Fuck now I gotta redo the gibtonite/bananium/iron

File: code/game/turfs/simulated/floor/plating.dm [hunk 1]
```diff
@@ -239,6 +236,27 @@
 			L.adjust_fire_stacks(20)
 			L.IgniteMob()
 
+/turf/simulated/floor/plating/lava/attackby(obj/item/C, mob/user, params) //Lava isn't a good foundation to build on
```
Comment (Razharas) [inline code/game/turfs/simulated/floor/plating.dm]: >Floor plating lava

Im out

**Comment by KorPhaeron** [comment]
@Razharas

That as already reviewed and approved when sticky did it, then again when I revived his PR

What is wrong with it?

**Comment by Anonus** [comment]
There's no good reason to have lava be a child of plating, it could just be a child of floor or even a child of simulated.

**Comment by Dorsisdwarf** [comment]
Presumably its so you can build a bridge over troubled lava

**Comment by Anonus** [comment]
You can't effectively build a bridge over lava anyway, though, you have to build a wall on it, then remove the wall and that takes a ton of time, in addition to being super fucking unintuitive.

**Comment by Bawhoppen** [comment]
I don't think you were supposed to merge this yet

**Pull request merged**